internal/apiserver/ffi2swagger.go

```diff
@@ -73,34 +77,45 @@ func (og *ffiSwaggerGen) Generate(ctx context.Context, baseURL string, api *core
 	}
 
 	return ffapi.NewSwaggerGen(&ffapi.Options{
-		Title:       ffi.Name,
-		Version:     ffi.Version,
-		Description: ffi.Description,
-		BaseURL:     baseURL,
+		Title:                 ffi.Name,
+		Version:               ffi.Version,
+		Description:           ffi.Description,
+		BaseURL:               baseURL,
+		DefaultRequestTimeout: config.GetDuration(coreconfig.APIRequestTimeout),
 	}).Generate(ctx, routes)
 }
 
 func (og *ffiSwaggerGen) addMethod(routes []*ffapi.Route, method *core.FFIMethod, hasLocation bool) []*ffapi.Route {
+	description := method.Description
+	if method.Details != nil && len(method.Details) > 0 {
+		description = fmt.Sprintf("%s\n\nAdditional smart contract details:\n\n%s", description, buildDetailsTable(method.Details))
```

Comment:
Struggling a little to understand this. What is the end to end of the `details`? Is it the place where `devdocs` would be specified?

One specific item (along with the wider question) is the `Additional smart contract details` string needs to be translated.

Reply:
Sure! I did forget to mention that in the description, sorry about that. The short version is that it's a way to bubble up blockchain specifics about a function, like if it's `payable` or not.

Good call out on the string translation, thanks.